apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -53,17 +77,15 @@ var ApplabVisualizationColumn = React.createClass({
             screenIds={this.props.screenIds}
             onScreenCreate={this.props.onScreenCreate} />
         }
-        <PhoneFrame
-            showFrame={showFrame}
-            isDark={this.props.isRunning}
-            showSelector={this.props.interfaceMode === applabConstants.ApplabInterfaceMode.DESIGN}
-            screenIds={this.props.screenIds}
-            onScreenCreate={this.props.onScreenCreate}
-        >
-          <Visualization/>
-        </PhoneFrame>
-        <GameButtons instructionsInTopPane={this.props.instructionsInTopPane}>
-          <CompletionButton/>
+        {visualization}
+        <GameButtons>
```

Comment:
TODO - brent  add a comment explaining that this div controls where or not button is centered